README.md

```diff
@@ -0,0 +1,100 @@
+# sql_insert_writer
+
+[![PyPI Status](https://img.shields.io/pypi/v/sql_insert_writer.svg)](https://pypi.python.org/pypi/sql_insert_writer)
+[![CircleCI](https://circleci.com/gh/18F/sql_insert_writer.svg?style=svg)](https://circleci.com/gh/18F/sql_insert_writer)
+[![Code Climate](https://codeclimate.com/github/18F/sql_insert_writer/badges/gpa.svg)](https://codeclimate.com/github/18F/sql_insert_writer)
+[![Test Coverage](https://codeclimate.com/github/18F/sql_insert_writer/badges/coverage.svg)](https://codeclimate.com/github/18F/sql_insert_writer/coverage)
+[![Dependency Status](https://gemnasium.com/badges/github.com/18F/sql_insert_writer.svg)](https://gemnasium.com/github.com/18F/sql_insert_writer)
+
+Helps generate highly readable SQL INSERT statements
+
+```
+$ sql_insert_writer pet
+
+INSERT INTO pet (
+  id,
+  name,
+  species_name,
+  planet,
+  kg
+)
+VALUES
+(
+  DEFAULT,  -- ==> id
+  DEFAULT,  -- ==> name
+  DEFAULT,  -- ==> species_name
+  DEFAULT,  -- ==> planet
+  DEFAULT  -- ==> kg
+)
+
+$ sql_insert_writer pet animal
+
+INSERT INTO pet (
+  id,
+  name,
+  species_name,
+  planet,
+  kg
+)
+SELECT
+  id,  -- ==> id
+  name,  -- ==> name
+  species_name,  -- ==> species_name
+  planet,  -- ==> planet
+  DEFAULT  -- ==> kg
+FROM animal
+```
+
+* Documentation: https://sql-insert-writer.readthedocs.io.
```

Comment:
Aw, this gives me a 404 :(

Reply:
Oops, yes, I need to remove that!  I decided the whole sphinx / read-the-docs treatment would be overkill for this.